meetings/eng/2022.md

```diff
@@ -1,5 +1,102 @@
 # 2022 Meeting Notes
 
+## 2022-05
+
+- Facilitator: Sarah
+- Notetaker: Erik, Jim
+
+### Job hire update/decision making/discussion
+
+- [Proposal document is here](https://docs.google.com/document/d/1PHpLJniwOIWXrhUcEiFhnOwPVQC9NSkoyHQeDss4udg/edit?usp=sharing)
+  - Brief description of current candidates
+  - Describe suggestion to make an offer
+  - Question: Any objections to making the offer?
+  - Discuss potential second hire
+  - Question: Worth pursuing if we have budget?
+
+Make an offer:
+
+- CH: meeting goal: to get a feeling one way or another to provide a job offer for a candidate.
+- Important that we document the process we used.
+- DA: We have consensus on hiring top candidate? +1s all around...
+
+Additional option:
+
+- CH: option? Possibility to hire second candidate as well? Worthwhile? Complimentarity of two candidates?
+- DA: close to what CH described. second candidate is focused on community; technical knowledge. top candidate may be more focused on the product side; Both top candidate and second candidate can do both roles with top candidate seen as having strength across both dimensions of the Product/Community lead.
+- DA: strategically it would be great to hire both! Are we covering our bases well? Do we need another person like the P/C Lead? Or do we need more personnel for support. Funding? Resources? We spent a lot of time searching for good people. We found two!
+- CH: Agree with your point about support. Do these people fit into the support role? They could provide this capacity. DA: closer to community so they may filter out some of the requests and lower the support volume on engineering.
+- SG: top candidate's plan to deploy community support infrastructure will likely reduce the support volume by enabling community-sourced responses to support needs. If money is available, we should hire. Can we afford to spend this much time on hiring in the future?
+- GD: Be mindful of second candidate' time. We need to make offers in a timely manner. CH: Agree! We can be transparent about the status. "Don't have the money now but we want to hire you, etc."
```

Comment:
```suggestion
```

IMO this is all probably too sensitive to put into public notes...instead I think we should copy/paste these into a discussion doc in the hiring folder for this position.